### PR TITLE
FUSETOOLS2-226 - Configure code coverage

### PR DESCRIPTION
**Pull request by apupier**
Signed-off-by: Aurélien Pupier <apupier@redhat.com>

### CODE REVIEW

**Comment by apupier** [comment]
running locally sonar-scanner reporting the coverage:
Screenshot from 2020-02-14 16-23-30

File: coverconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "enabled": true,
+    "relativeSourcePath": "../../../out",
```
Comment (apupier) [inline coverconfig.json]: here is the difference with vscod-tekton configuration. They are pointing to src but we are generating js file sin out folder.

**Comment by bfitzpat** [comment]
Interesting. Is this the next level of sonar feedback on code quality?

**Comment by apupier** [comment]
> s this the next level of sonar feedback on code quality?

I don't understand what you're asking for. Can you precise please?
This Pull Request allows to have code coverage report on classic Mocha tests using Istanbul. I [configured Sonar](https://github.com/camel-tooling/vscode-camelk/pull/307/files#diff-c417eb9ba61d2be90ac7248a85347acaR6) to be able to read this code coverage report.

**Pull request merged**